docs/architecture.rst

```diff
@@ -57,6 +57,20 @@ The key-value store is implemented by a grid of Tahoe-LAFS storage servers --
 user-space processes. Tahoe-LAFS storage clients communicate with the storage
 servers over TCP.
 
+There are two supported protocols:
+
+* Foolscap, the only supported protocol in release before v1.19.
+* HTTPS, new in v1.19.
+
+By default HTTPS is disabled (this will change in
+https://tahoe-lafs.org/trac/tahoe-lafs/ticket/4041). When HTTPS is enabled on
+the server, the server transparently listens for both Foolscap and HTTP on the
```

Comment:
Mixing "HTTP" and "HTTPS in this section might be confusing.  Maybe we should always refer to HTTPS?